plugins/rbac-backend/src/service/permission-policy.ts

```diff
@@ -166,6 +172,13 @@ export class RBACPermissionPolicy implements PermissionPolicy {
       'permission.rbac.policyFileReload',
     );
 
+    if (superUsers && superUsers.length > 0) {
+      for (const user of superUsers) {
+        const userName = user.getString('name');
```

Comment:
It would be nice to add validation `validateEntityReference` before putting userName to the list

Reply:
We have such validation for admins.

Reply:
Oh, never mind we have validation bottom